docs/usage/validation_errors.md

```diff
@@ -1,14 +1,14 @@
-Below are details on common validation errors users may encounter when working with pydantic, together
-with some suggestions on how to fix them.
+Pydantic attempts to provide useful validation errors. Below are details on common validation errors users
+may encounter when working with pydantic, together with some suggestions on how to fix them.
 
 ## `arguments_type`
 
 This error is raised when arguments passed to a function are not
-a tuple, list, or a dictionary.
+a `Tuple`, `List`, or dictionary.
```

Comment:
```suggestion
a tuple, list, or dictionary.
```
I think this is okay since these are terms of art in python world. I could possibly be convinced to use this instead:
```suggestion
a `tuple`, `list`, or `dict`.
```
but I think the original is better (minus the double 'a').